archunit/src/main/java/com/tngtech/archunit/lang/conditions/ArchConditions.java

```diff
@@ -775,91 +775,145 @@ public static AllDependenciesCondition onlyHaveDependenciesWhere(DescribedPredic
         return not(ArchConditions.<HAS_ANNOTATIONS>beMetaAnnotatedWith(predicate));
     }
 
+    /**
+     * @return A condition matching classes analogously to {@link JavaClass.Predicates#implement(Class)}
```

Comment:
I wouldn't start with a capital `A` here (and in the similar cases below).
```suggestion
     * @return a condition matching classes analogously to {@link JavaClass.Predicates#implement(Class)}
```